site/layouts/default.html

```diff
@@ -29,12 +29,12 @@
           <h3><%= link_to "Readme", "/" %></h3>
           <h3>Guides</h3>
           <ul>
-            <li><%= link_to "Introduction", "/graphql-ruby/guides/introduction/" %></li>
-            <li><%= link_to "Defining a Schema", "/graphql-ruby/guides/defining_your_schema/" %></li>
-            <li><%= link_to "Executing Queries", "/graphql-ruby/guides/executing_queries/" %></li>
-            <li><%= link_to "Code Reuse", "/graphql-ruby/guides/code_reuse/" %></li>
-            <li><%= link_to "Security", "/graphql-ruby/guides/security/" %></li>
-            <li><%= link_to "Testing", "/graphql-ruby/guides/testing/" %></li>
+            <li><%= link_to "Introduction", "/guides/introduction/" %></li>
+            <li><%= link_to "Defining a Schema", "/guides/defining_your_schema/" %></li>
+            <li><%= link_to "Executing Queries", "/guides/executing_queries/" %></li>
+            <li><%= link_to "Code Reuse", "/guides/code_reuse/" %></li>
+            <li><%= link_to "Security", "/guides/security/" %></li>
+            <li><%= link_to "Testing", "/guides/testing/" %></li>
```

Comment:
I added `graphql-ruby/` to the paths because this was getting deployed to `rmosolgo.github.com/graphql-ruby/`, is that prefix being added another way?